<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changelog</class>
 <widget class="QWidget" name="changelog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>341</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {	background-color: #5A5A5A; }

QPushButton {
	color: #ffffff;
	border: none;
	padding: 5px;
	background-color: #3a3a3a;
	border-radius: 5px;
}

QPushButton::pressed { background-color: #2d5bd4; }</string>
  </property>
  <widget class="QPushButton" name="chlog_close_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_desc_lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>55</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 97, 167);</string>
   </property>
   <property name="text">
    <string>Ayckinn's Password Generator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_lbl">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>0</y>
     <width>345</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 97, 167);</string>
   </property>
   <property name="text">
    <string>GENPASS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="chlog_version_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Version 1.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="chlog_line_1_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg_lbl">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>105</y>
     <width>331</width>
     <height>77</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="chlog_line_2_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="chlog_line_3_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>title_lbl</zorder>
  <zorder>bg_lbl</zorder>
  <zorder>chlog_close_btn</zorder>
  <zorder>title_desc_lbl</zorder>
  <zorder>chlog_version_lbl</zorder>
  <zorder>chlog_line_1_lbl</zorder>
  <zorder>chlog_line_2_lbl</zorder>
  <zorder>chlog_line_3_lbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
